touchforms/formplayer/static/formplayer/script/fullform-ui.templates.js

```diff
@@ -0,0 +1,48 @@
+this["JST"] = this["JST"] || {};
+
+this["JST"]["formplayer/templates/formplayer/fullform-ui/form.html"] = function(obj) {
+obj || (obj = {});
+var __t, __p = '', __e = _.escape;
+with (obj) {
+__p += '<div>\n    <h1 id="title"></h1>\n    <div id="form"></div>\n    <input id="submit" type="submit" value="Submit" />\n</div>\n';
```

Comment:
no, but we don't currently have a build process setup on the server for it. so for now it has to be in the repo in order for things to work